root/etc/e-smith/events/actions/nethserver-tancredi-conf

```diff
@@ -11,3 +11,50 @@ GRANT SELECT on asterisk.ampusers TO 'tancredi'@'localhost' identified by '$db_p
 FLUSH privileges;
 EOF
 
+# set default.ini on first install
+dst_file="/var/lib/tancredi/data/scopes/defaults.ini"
+
+if [[ ! -f ${dst_file} ]]; then
+    echo "NOTICE: $0 initializing ${dst_file}"
+    /bin/cp -v /usr/share/tancredi/data/scopes/defaults.ini ${dst_file}
+
+    # Add variable for UI first configuration
+    echo 'ui_first_config = "1"' >> ${dst_file}
+
+    # Remove variables if they exists
+    for variable in timezone language tonezone hostname provisioning_url_scheme time_format date_format ldap_server ldap_port ldap_tls ldap_user ldap_password ldap_base ldap_name_display ldap_number_attr ldap_mainphone_number_attr ldap_mobilephone_number_attr ldap_otherphone_number_attr ldap_name_attr ldap_number_filter ldap_name_filter adminpw userpw; do
+        sed -i '/^'${variable}' =.*/d' ${dst_file}
+    done
+
+    # Add defaults
+    echo 'timezone = "Europe/Rome"' >> ${dst_file}
+    echo 'language = "it"' >> ${dst_file}
+    echo 'tonezone = "it"' >> ${dst_file}
+    echo 'hostname = "'$(hostname)'"' >> ${dst_file}
+    echo 'provisioning_url_scheme = "https"' >> ${dst_file}
+
+    # Add time_format and date_format
+    echo 'time_format = "24"' >> ${dst_file}
+    echo 'date_format = "DD/MM/YYYY"' >> ${dst_file}
+
+    # Add ldap defaults
+    echo 'ldap_server = "'$(hostname)'"' >> ${dst_file}
+    echo 'ldap_port = "10389"' >> ${dst_file}
+    echo 'ldap_tls = "none"' >> ${dst_file}
+    echo 'ldap_user = ""' >> ${dst_file}
+    echo 'ldap_password = ""' >> ${dst_file}
+    echo 'ldap_base = ""' >> ${dst_file}
```

Comment:
Are you sure it can work without any ldap_base? In my experience with OpenLDAP  the ldap_base is required.

Reply:
you're right, default should be "dc=phonebook,dc=nh"